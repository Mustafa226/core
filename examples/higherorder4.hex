% test higher order creating constant input for external atom
p(a).
p(b).

pred1(in).
pred2(p).

P(X) :- pred1(P), pred2(X). 

out(Y) :- &testConcat[x,X,x](Y), in(X).
