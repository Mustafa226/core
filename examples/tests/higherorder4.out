{in(a),in(b),out(xax),out(xbx)}
